### ISSUE TITLE
UI suggestions for SSVC-calc

### ISSUE THREAD

**Issue by ahouseholder** [CERTCC/SSVC#438]
Capturing feedback from @bkoo in #412:

> I mostly echo what @ehatleback has already mentioned with an emphasis on his second point. By having the selections always visible to enable visual continuity, it also will help see the final decision point as both modes (Graphic/Analyst) block the final result with an export window. At the very least, perhaps move the export window out of the way so that the result can be seen? 
<img width="1132" alt="Screenshot 2024-01-31 at 5 48 00 PM" src="https://github.com/CERTCC/SSVC/assets/111892109/b0cde54d-4737-48f3-beb1-f7679925d8ea">
<img width="1139" alt="Screenshot 2024-01-31 at 5 48 20 PM" src="https://github.com/CERTCC/SSVC/assets/111892109/8cb02b68-9e7d-4d90-9c92-17b9fe17d4a8">

_Originally posted by @sei-bkoo in https://github.com/CERTCC/SSVC/issues/412#issuecomment-1920124439_
            

I'm creating this as a separate issue from #412 since it covers new changes and #412 is just about making sure the existing calculator works in the new layout.

**Comment by ahouseholder** [comment]
also capturing @ehatleback's [comments](https://github.com/CERTCC/SSVC/issues/412#issuecomment-1919764611) in #412:

> In the first decision point, scrolling over the options highlights the top and bottom options (None and Active) in red, but the middle option (PoC) does not include the red highlighting.
>
> Additionally, I'd suggest that the appearance of the options when I hover is somehow a bit jarring or disorienting. A better option might be for the available selections to either be available all the time on the screen (rather than appearing when you hover), or to appear when you hover, but always to appear in the same location on the screen, like at the bottom.
>
> Finally, the options that appear when you hover frequently look like they're a little bit too big for the window they appear in; the text is right at the very edge of, and sometimes even cut off a bit from, the box that contains the appearing text.

**Comment by ahouseholder** [comment]
It is not clear to me whether this issue was fully or only partially resolved by 

- #513

Also note there are some more possibly related comments in #514. Both #514 and #438 should be revised into separately resolvable issues if possible.

**Comment by sei-vsarvepalli** [comment]
Yep only partially resolved the Export window issue. The other appearance related ones remain.

**Comment by sei-vsarvepalli** [comment]
I think this is resolved now. At least both these issues are solved now.
